lib/api-export.js

```diff
@@ -15,6 +15,10 @@ class ApiExport extends ApiSaveData {
 		return exportStruct;
 	}
 
+	get statuses() {
```

Comment:
Pregunta: 

Es necesario hacer este getter ? 
No se puede usar directamente : `this.model.constructor.statuses.pending` ?

Reply:
Si bien funciona igual ponerlo directamente me parece que queda más prolijo y fácil de leer usando el getter que ya separa el model.constructor.statuses, se usa tantas veces los statuses que queda muy largo